### ISSUE TITLE
OLM files are not pushed during automated release of Che Operator

### ISSUE THREAD

**Issue by mkuznyetsov** [eclipse-che/che#18393]
### Describe the bug
When releasing operator through GitHub Actions workflow, the stage when OLM files are supposed to be pushed to Quay.io application doesn't work:

```
ERROR:operatorcourier.push:{"error":{"code":"forbidden","details":{"package":"eclipse-che-operator-kubernetes/eclipse-che-preview-kubernetes","scopes":["push"]},"message":"Unauthorized access for: eclipse-che-operator-kubernetes/eclipse-che-preview-kubernetes"}}

Unauthorized access for: eclipse-che-operator-kubernetes/eclipse-che-preview-kubernetes
```

Special robot account credential secrets are being used in QUAY_USERNAME_OS, QUAY_PASSWORD_OS, QUAY_USERNAME_K8S, QUAY_PASSWORD_K8S secrets, which are working if run from local machine.

### Steps to reproduce

Run"release" workflow in che-operator repo, with `./make-release.sh <VERSION> --push-olm-files` part enabled

### Expected behavior
OLM files are pushed to quay applications:
https://quay.io/application/eclipse-che-operator-openshift/eclipse-che-preview-openshift
https://quay.io/application/eclipse-che-operator-kubernetes/eclipse-che-preview-kubernetes

**Comment by nickboldt** [comment]
@tolusha is it possible that the secrets we need in the https://github.com/eclipse/che-operator repo are not set up correctly?

```
          export QUAY_USERNAME_K8S=${{ secrets.QUAY_ECLIPSE_CHE_OPERATOR_KUBERNETES_USERNAME }}
          export QUAY_PASSWORD_K8S=${{ secrets.QUAY_ECLIPSE_CHE_OPERATOR_KUBERNETES_PASSWORD }}
          export QUAY_USERNAME_OS=${{ secrets.QUAY_ECLIPSE_CHE_OPERATOR_OPENSHIFT_USERNAME }}
          export QUAY_PASSWORD_OS=${{ secrets.QUAY_ECLIPSE_CHE_OPERATOR_OPENSHIFT_PASSWORD }}
```

I can't check https://github.com/eclipse/che-operator/settings/secrets/actions so I don't know what's in there...

Or, is is possible that the credentials stored there are NOT configured in Quay to be able to push to application repos? 

Can we take some time this week to investigate this? I'd love to get this step automated in time for 7.23 so we have one less manual thing to do after generating pull requests for the new release.

**Comment by nickboldt** [comment]
Suggestion from Flavius is to use same user/pwd setup as in https://github.com/eclipse/che-operator/blob/master/.github/workflows/operator-build-nightly.yaml#L30-L32:

```
export QUAY_USERNAME_K8S=${{ secrets.QUAY_USERNAME }}
export QUAY_PASSWORD_K8S=${{ secrets.QUAY_PASSWORD }}
export QUAY_USERNAME_OS=${{ secrets.QUAY_USERNAME }}
export QUAY_PASSWORD_OS=${{ secrets.QUAY_PASSWORD }}
```

**Comment by nickboldt** [comment]
Same error:

```
Fri, 04 Dec 2020 15:09:47 GMT
   - Push flattened files to Quay.io namespace 'eclipse-che-operator-kubernetes' as version 9.9.1607094587+7c4d6955
Fri, 04 Dec 2020 15:10:08 GMT
ERROR:operatorcourier.push:{"error":{"code":"forbidden","details":{"package":"eclipse-che-operator-kubernetes/eclipse-che-preview-kubernetes","scopes":["push"]},"message":"Unauthorized access for: eclipse-che-operator-kubernetes/eclipse-che-preview-kubernetes"}}
Fri, 04 Dec 2020 15:10:08 GMT

Fri, 04 Dec 2020 15:10:08 GMT
Unauthorized access for: eclipse-che-operator-kubernetes/eclipse-che-preview-kubernetes
Fri, 04 Dec 2020 15:10:08 GMT
Error: Process completed with exit code 1.
```

when running gh action from https://github.com/eclipse/che-operator/pull/566 (https://github.com/eclipse/che-operator/tree/fix-18393 ) using version = 7.22.0

**Comment by nickboldt** [comment]
Fixed with https://github.com/eclipse/che-operator/pull/566
